bin/process_serotyper_results.py

```diff
@@ -26,14 +16,20 @@ def make_gene_dict(input_file, depth_threshold):
 
 def write_outfile(gene_dict, out_file):
     """Write serotype, match type status and average read depth to output file"""
+    matched_alleles = []
+    match_type = []
+    serotype = []
+    avgdepth = []
+    for key, values in gene_dict.items():
+        status = 'identical'
+        if values[4] != '':
+            status = 'imperfect'
```

Comment:
You probably know this, so only in case you don't: The highlighted code could be a one-liner:
`status = `'identical' if values[4] == ''" else 'imperfect'